### PR TITLE
Swift65 Hotswap Support

### PR DESCRIPTION
**Pull request by spooknik**
Add Support for Swift65 Hotswap

## Description

Very similar to [solder version](https://github.com/qmk/qmk_firmware/tree/master/keyboards/alfredslab/swift65/solder) which is already in the repo. Biggest differences are the keymap. I'm not a pro in C, so if there is something wrong just tell me and i'll fix it best I can 👍 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [] My change requires a change to the documentation.
- [] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/alfredslab/swift65/hotswap/config.h [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+// /*== or choose animations ==*/
+//     #define RGBLIGHT_EFFECT_BREATHING
+//     #define RGBLIGHT_EFFECT_RAINBOW_MOOD
+//     #define RGBLIGHT_EFFECT_RAINBOW_SWIRL
+//     #define RGBLIGHT_EFFECT_SNAKE
+//     #define RGBLIGHT_EFFECT_KNIGHT
+//     #define RGBLIGHT_EFFECT_CHRISTMAS
+//     #define RGBLIGHT_EFFECT_STATIC_GRADIENT
+//     #define RGBLIGHT_EFFECT_RGB_TEST
+//     #define RGBLIGHT_EFFECT_ALTERNATING
```
Comment (drashna) [inline keyboards/alfredslab/swift65/hotswap/config.h:66]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

File: keyboards/alfredslab/swift65/hotswap/info.json [hunk 2]
```diff
@@ -0,0 +1,83 @@
+{
+    "keyboard_name": "hotswap",
+    "url": "https://swiftcables.net/",
+    "maintainer": "Spooknik",
+    "layout_aliases": {
+        "LAYOUT": "LAYOUT_7u_space",
+        "LAYOUT_all": "LAYOUT_7u_space"
+        },
+    "width": 16.25,
+    "height": 5,
```
Comment (drashna) [inline keyboards/alfredslab/swift65/hotswap/info.json:10]:
```suggestion
```

File: keyboards/alfredslab/swift65/hotswap/rules.mk [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+#     If no clock division is performed on the input clock inside the AVR (via the
+#     CPU clock adjust registers or the clock division fuses), this will be equal to F_CPU.
+F_USB = $(F_CPU)
+
+# Interrupt driven control endpoint task(+60)
+OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
+
+
+# Boot Section Size in *bytes*
+OPT_DEFS += -DBOOTLOADER_SIZE=4096
```
Comment (drashna) [inline keyboards/alfredslab/swift65/hotswap/rules.mk:41]:
```suggestion
# Bootloader selection
BOOTLOADER = atmel-dfu
```

File: keyboards/alfredslab/swift65/hotswap/rules.mk [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+BOOTMAGIC_ENABLE ?= yes	# Virtual DIP switch configuration(+1000)
+MOUSEKEY_ENABLE ?= yes	# Mouse keys(+4700)
+EXTRAKEY_ENABLE ?= yes	# Audio control and System control(+450)
+CONSOLE_ENABLE ?= no	# Console for debug(+400)
+COMMAND_ENABLE ?= no    # Commands for debug and configuration
+SLEEP_LED_ENABLE ?= no  # Breathing sleep LED during USB suspend
+NKRO_ENABLE ?= yes		# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE ?= no  # Enable keyboard backlight functionality
+AUDIO_ENABLE ?= no
+RGBLIGHT_ENABLE ?= yes
```
Comment (drashna) [inline keyboards/alfredslab/swift65/hotswap/rules.mk:56]:
```suggestion
# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
NKRO_ENABLE = yes           # Enable N-Key Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by spooknik** [comment]
Thanks for the suggestions, I committed them all.

**Review by drashna — APPROVED** [review]

**Comment by spooknik** [comment]
Hello again! Could someone else sign off on this request? Not to be pushy but it's been about a month and no one else has chimed in.

**Review by tzarc — APPROVED** [review]

**Pull request merged**